pkg/cli/admin/release/extract_tools.go

```diff
@@ -448,6 +448,18 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			InjectReleaseVersion: true,
 			ArchiveFormat:        "openshift-baremetal-install-linux-%s.tar.gz",
 		},
+		{
+			OS:       "linux",
+			Arch:     targetReleaseArch,
+			Command:  "openshift-install-fips",
+			Optional: true,
+			Mapping:  extract.Mapping{Image: "baremetal-installer", From: "usr/bin/openshift-install"},
+
+			Readme:               readmeInstallUnix,
+			InjectReleaseImage:   true,
+			InjectReleaseVersion: true,
+			ArchiveFormat:        "openshift-baremetal-install-linux-%s.tar.gz",
```

Comment:
Since this target's tarball name is same with the above one, there might be unexpected binary overrides. But I verified that `--tools` flag ignores the targets that are marked as Optional, so that these ArchiveFormats are not used at all and we are good.

Reply:
Thanks, I updated this one anyway.